Jenkinsfile

```diff
@@ -35,7 +35,7 @@ pipeline {
                 		
                         stage('Integration tests') {
                 			steps {
-                				sh 'mvn verify -B -Pintegration-tests -Dwildfly-14 -DskipTests=true -Dmaven.test.failure.ignore=true'
+                				sh 'mvn verify -B -Pintegration-tests -Pwildfly-14 -DskipTests=true -Dmaven.test.failure.ignore=true'
```

Comment:
What's the reason to run wildfly 14 for integration tests? 
* Shouldn't we run the latest aka wildfly16?
* Shouldn't we define a default which would be the latest wildfly aka 16?
* Shouldn't we defined an alternative that runs with jdk8 aka wildfly 10 (see https://github.com/redhat-developer/rsp-server/blob/master/runtimes/tests/org.jboss.tools.rsp.server.wildfly.test/src/main/java/org/jboss/tools/rsp/server/wildfly/test/servertype/ServerJavaVersionTest.java#L149)?

Reply:
Ad 1: We run wildfly 16 in java 11 branch.
Ad 2: How do you do that? I do not see a way when using maven profiles unless we download wf 16 by default even if we use and download another wf server. 
Ad 3: I believe that we will end up there, running full suite of server under all configurations of OSes and java platforms... But in near future, right?

Reply:
Ad 3-2: wildfly 10 for testing java 1.8 compatibility seems to me like testing quite old stuff, I would prefer latest supporting mainly 1.8?

Reply:
> Ad 3-2: wildfly 10 for testing java 1.8 compatibility seems to me like testing quite old stuff, I would prefer latest supporting mainly 1.8?

I just didnt understand why we are running wfl14 when there's jdk8. At least that's how I understand the jenkins file? Wrong?

> Ad 1: We run wildfly 16 in java 11 branch.

Ok for that. On the other hand, with jdk18, aren't we running wfl14? If that's the case (correct me if I'm wrong, please) I dont see why. Wildf16 also runs with jdk8, is latest as thus should also run when testing jdk18 I think.

> Ad 2: How do you do that? I do not see a way when using maven profiles unless we download wf 16 by default even if we use and download another wf server.

```   
<activation>
        <activeByDefault>true</activeByDefault>
</activation>
```
You should end up running wfl16 aka latest if no profile is given in cmd-line. Missing something?

> Ad 3: I believe that we will end up there, running full suite of server under all configurations of OSes and java platforms... But in near future, right?

I was guessing that we're runnging wfl14 when we have jdk18. I didnt see why we would do that given that Wildfly16 also runs with jdk8 (see above)

Reply:
> > Ad 3-2: wildfly 10 for testing java 1.8 compatibility seems to me like testing quite old stuff, I would prefer latest supporting mainly 1.8?
> 
> I just didnt understand why we are running wfl14 when there's jdk8. At least that's how I understand the jenkins file? Wrong?
> 
> > Ad 1: We run wildfly 16 in java 11 branch.
> 
> Ok for that. On the other hand, with jdk18, aren't we running wfl14? If that's the case (correct me if I'm wrong, please) I dont see why. Wildf16 also runs with jdk8, is latest as thus should also run when testing jdk18 I think.
> 
> > Ad 2: How do you do that? I do not see a way when using maven profiles unless we download wf 16 by default even if we use and download another wf server.
> 
> ```
> <activation>
>         <activeByDefault>true</activeByDefault>
> </activation>
> ```
> 
> You should end up running wfl16 aka latest if no profile is given in cmd-line. Missing something?
Nope, that does not work when we always run integration-tests profile. But I got an idea, I will make new push into PR. Seems like it is working.
> 
> > Ad 3: I believe that we will end up there, running full suite of server under all configurations of OSes and java platforms... But in near future, right?
> 
> I was guessing that we're runnging wfl14 when we have jdk18. I didnt see why we would do that given that Wildfly16 also runs with jdk8 (see above)
I believe we can use wf 16 on both javas, no problem with that, wf-14 could have brought just another piece of server runtime to test on. But since we plan to have tests for full matrix of runtimes and another configs, we do no really need to have it here.